partitioned-heat-conduction/solver-fenics/heat_pySDC_problem_class.py

```diff
@@ -0,0 +1,130 @@
+from fenics import *
+import dolfin as df
```

Comment:
```suggestion
import fenics
```
Generally, I would suggest to avoid wildcard imports. See https://stackoverflow.com/questions/3615125/should-wildcard-import-be-avoided.

Do you really need to import `dolfin`? Usually all the relevant functions from `dolfin` are also in `fenics` (which is to my understanding the high-level wrapper).

What I usually do:
```
from fenics import function_that_I_use, another_function_that_I_use, ...
```
Then you also do not have to bother with all the `fenics.some_fenics_function_I_use_but_did_not_import(...)` below and just call `some_fenics_function_I_imported(...)`

Reply:
Fixed